Xamarin.Forms.Platform.UAP/PickerRenderer.cs

```diff
@@ -176,11 +176,6 @@ void UpdateCharacterSpacing()
 		{
 			Control.CharacterSpacing = Element.CharacterSpacing.ToEm();
 
-			if (Control.Header is TextBlock header)
```

Comment:
Maybe I'm not understanding exactly what is going on here, but where did the character spacing of the header move to now?

Reply:
Never mind, I see I missed the discussion about that :D